tasks/git.py

```diff
@@ -26,3 +28,18 @@ def check_protected_branch(ctx):
             )
         )
         raise Exit(code=1)
+
+
+@task
+def push_signed_commits(ctx: Context, branch: str, commit_message: str, source_branch: str | None = None) -> None:
+    """Create a tree from local stage changes, commit and push using API to get signed commits from bots.
+
+    Keywords arguments:
+    ctx -- Invoke context
+    update_branch -- The branch to push to
+    commit_message -- The commit message to use
```

Comment:
Nit:
```suggestion
    Args:
        ctx: Invoke context
        update_branch: The branch to push to
        commit_message: The commit message to use
```
I prefer using the google python style